src/code42cli/cmds/search/extraction.py

```diff
@@ -8,14 +8,26 @@
 import code42cli.errors as errors
 from code42cli.date_helper import verify_timestamp_order
 from code42cli.logger import get_main_cli_logger
-from code42cli.output_formats import get_dynamic_header
+from code42cli.output_formats import to_table
 from code42cli.util import warn_interrupt
 
 logger = get_main_cli_logger()
 
 _ALERT_DETAIL_BATCH_SIZE = 100
 
 
+def handle_include_all(include_all, default_header, format_func):
+    output_header = None
+    if format_func.__name__ == to_table.__name__:
```

Comment:
if we want to do away with using `__name__`, we can get rid of [callback](https://github.com/code42/code42cli/pull/138/files#r473037679) argument and define another method that maps the function to the options.

Just an alternative suggestion.

Reply:
I will try this

Reply:
I did this